frontend/packages/console-app/src/components/cloud-shell/__tests__/CloudShellTab.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { mount } from 'enzyme';
+import { Provider } from 'react-redux';
+import store from '@console/internal/redux';
+import CloudShellTab from '../CloudShellTab';
+import CloudShellTerminal from '../CloudShellTerminal';
+
+describe('CloudShell Tab Test', () => {
+  const cloudShellTabWrapper = mount(<CloudShellTab />, {
+    wrappingComponent: ({ children }) => <Provider store={store}>{children}</Provider>,
+  });
+  it('should render correct title', () => {
+    expect(cloudShellTabWrapper.find('[data-test-id="terminal-tab-header"]').exists()).toBe(true);
+    expect(cloudShellTabWrapper.find('[data-test-id="terminal-tab-header"]').text()).toEqual(
+      'Openshift command Line Terminal',
+    );
+  });
+  // it('should render restore button', () => {
```

Comment:
Remove comments.

Reply:
All the commented out portions are to be brought back.. these are being temporarily hidden till we work on the session thing.

Reply:
I'll update the comments with the note

Reply:
Well, I would prefer no code instead of commented out code anyways. You can easily go back in git history and get the code back when needed.

Reply:
Updated this